tests/Makefile

```diff
@@ -2,7 +2,7 @@
 # To compile on SunOS: add "-lsocket -lnsl" to LDFLAGS
 # To compile with PKCS11: add "-lpkcs11-helper" to LDFLAGS
 
-CFLAGS	?= -O2
+CFLAGS	?= -g3 #-O2
```

Comment:
This line looks like a mistake. We've turned on debugging with `-g3` which is fair in test code but we've commented out optimisations.

Why comment them out? Either delete them, or leave them in. Commented out code is bad practice.